README.md

```diff
@@ -1,57 +1,88 @@
+Select the `sensor` type, then select the `wifi_sensor:linux` model. 
+Enter a name for your component and click **Create**.
+
+> [!NOTE]  
+> For more information, see [Configure a Machine](https://docs.viam.com/manage/configuration/).
+
+### Attributes
+
+There are no attributes available for configuration with this service.
+
+### Example configuration
+
+```json
+{
+  "components": [
+    {
+      "name": "my-wifi-sensor",
+      "model": "viam:wifi_sensor:linux",
+      "type": "sensor",
+      "namespace": "rdk",
+      "attributes": {},
+      "depends_on": []
+    }
+  ],
+  "modules": [
+    {
+      "type": "registry",
+      "name": "viam_python-example-module",
+      "module_id": "viam:python-example-module",
+      "version": "0.0.13"
+    }
+  ]
+}
+```
+
+## Next steps
 
 Once the `wifi_sensor` has started collecting data, you can view reported readings from the [**Data** tab](https://app.viam.com/data/view?view=sensors) in the Viam app, under the **Sensors** subtab.
+
 You can also:
 
 - [Filter your results](https://docs.viam.com/data/view/#filter-data) using specific search criteria.
 - [Export your readings](https://docs.viam.com/data/export/) to your local workstation or another machine.
 - [Query your results](https://docs.viam.com/data/query/) using SQL or MQL directly in the App or from a MQL-compatible client.
+- [Visualize your results](https://docs.viam.com/data/visualize/) using popular third-party visualization platforms.
 - [Change the data capture frequency](https://docs.viam.com/data/capture/#configure-data-capture-for-individual-components) or [cloud sync frequency](https://docs.viam.com/data/cloud-sync/#configuration)
 
-## Writing a module
+## Learning resources
+
+### Writing a module
 
 This module includes basic module logic in its [`src/wifi_sensor.py`](https://github.com/viam-labs/python-example-module/blob/main/src/wifi_sensor.py) file.
 Follow the instructions to [create your own module](https://docs.viam.com/registry/create/) to learn how to write a custom module that extends a Viam API.
 
-### Setting up a `virtualenv`
+#### Setting up a `virtualenv`
 
 This module includes a `setup.sh` file to help configure a `virtualenv` for this module.
 Follow the instructions to [prepare your Python virtual environment](https://docs.viam.com/build/program/python-venv/) to learn how to set up and distribute a `virtualenv` configuration with your custom module.
 
-### Use CI to automatically publish on release
+#### Use CI to automatically publish on release
 
 This module includes a [`.github/workflows`](https://github.com/viam-labs/python-example-module/tree/main/.github/workflows) directory, which contains workflows that you can use to automatically publish a new version of your module to the Viam registry when you create a GitHub release.
 For more information, see [Update a module using a GitHub action](https://docs.viam.com/registry/upload/#update-an-existing-module-using-a-github-action).
 
-## Next Steps
```

Comment:
I think this is good info? Why remove

Reply:
ad least the first one with testing is helpful

Reply:
I moved Next steps up to line 56; the first sentence directs users to the Data tab to "test" (verify) that data has indeed been captured. Is that what you meant?

Reply:
Yep sorry